src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReferenceHandler.cs

```diff
@@ -10,6 +10,12 @@ namespace System.Text.Json.Serialization
     /// </summary>
     public abstract class ReferenceHandler
     {
+        /// <summary>
+        /// One of the enumeration values that indicates whether this ReferenceHandler implementation should use Preserve semantics or IgnoreCycle semantics.
```

Comment:
Right, there is also `ReferenceHandlingStrategy.None`, but that value should not be used by any resolver. I will take the suggestion since is more clear.